<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoPage</class>
 <widget class="QWidget" name="InfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>627</height>
   </rect>
  </property>
     <property name="locale">
         <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea{background: transparent; border: none}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
           <height>927</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{background: transparent}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>540</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">InfoLabel[type=&quot;error&quot;] {
    background-color: #FFCCCC;  /* Redish background for errors */
}

InfoLabel[type=&quot;warning&quot;] {
    background-color: #FFFFCC;  /* Yellowish background for warnings */
}

InfoLabel[type=&quot;information&quot;] {
    background-color: #CCE5FF;  /* Blueish background for information */
}

InfoLabel[type=&quot;success&quot;] {
    background-color: #CCFFCC;  /* Greenish background for success */
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QFrame" name="info_frame">
            <property name="styleSheet">
             <string notr="true">#info_frame[type=&quot;dark_Info&quot;] {
    background-color: rgb(39, 39, 39);
    border: 1px solid rgb(29, 29, 29);
    border-radius: 6px;
}

#info_frame[type=&quot;dark_Success&quot;] {
    background-color: rgb(57, 61, 27);
    border: 1px solid rgb(29, 29, 29);
    border-radius: 6px;
}

#info_frame[type=&quot;dark_Warning&quot;] {
    background-color: rgb(67, 53, 25);
    border: 1px solid rgb(29, 29, 29);
    border-radius: 6px;
}

#info_frame[type=&quot;dark_Error&quot;] {
    background-color: rgb(68, 39, 38);
    border: 1px solid rgb(29, 29, 29);
    border-radius: 6px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="ImageLabel" name="info_icon"/>
             </item>
             <item>
              <widget class="BodyLabel" name="info_text">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="SimpleCardWidget" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="BodyLabel" name="label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;关于 ImageColorVisualization&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;版本：&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1.0.0&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ImageColorVisualization 是一个图像颜色可视化工具，帮助用户分析和展示图像中的颜色分布。无论是设计师、摄影师还是数据分析师，都可以通过这个工具直观地理解图像的色彩组成，优化设计或进行色彩相关的研究。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;开发团队：&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ImageColorVisualization 团队&lt;br/&gt;© 2024 ImageColorVisualization 项目组 保留所有权利&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;许可证：&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;GNU General Public License v3.0 (GPL-3.0)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;特别感谢：&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;感谢各位编程工作者对开源社区的无私奉献！&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;用户反馈和支持：&lt;/span&gt;&lt;a href=&quot;mailto:support@imagecolorviz.com&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#cc5e29;&quot;&gt;support@imagecolorviz.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="SimpleCardWidget" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="BodyLabel" name="label_2">
               <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;
                       font-weight:700;&quot;&gt;Third-Party Libraries Used&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span
                       style=&quot; font-size:12pt;&quot;&gt;Thanks to the contributions of the following open-source
                       projects: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="3">
                 <widget class="ElevatedCardWidget" name="card_vtk">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="topMargin">
                    <number>50</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>50</number>
                   </property>
                   <item>
                    <widget class="QWidget" name="widget_5" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <spacer name="horizontalSpacer_5">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>61</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="ImageLabel" name="img_label_vtk">
                        <property name="pixmap">
                         <pixmap resource="../resource.qrc">:/logo/vtk</pixmap>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_6">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="StrongBodyLabel" name="label_6">
                     <property name="text">
                         <string notr="true">VTK</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="ElevatedCardWidget" name="card_qfw">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="topMargin">
                    <number>50</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>50</number>
                   </property>
                   <item>
                    <widget class="QWidget" name="widget_4" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <spacer name="horizontalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="ImageLabel" name="img_label_qfw">
                        <property name="pixmap">
                         <pixmap resource="../resource.qrc">:/logo/qfluentwidgets</pixmap>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_4">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>83</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="StrongBodyLabel" name="label_3">
                     <property name="text">
                         <string notr="true">QFluentWidgets</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignVCenter">
                 <widget class="ElevatedCardWidget" name="card_pyqt">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="topMargin">
                    <number>50</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>50</number>
                   </property>
                   <item>
                    <widget class="QWidget" name="widget_3" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="ImageLabel" name="img_label_pyqt">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="pixmap">
                         <pixmap resource="../resource.qrc">:/logo/pyqt</pixmap>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignmentFlag::AlignCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>69</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="StrongBodyLabel" name="label_4">
                     <property name="text">
                         <string notr="true">PySide6</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StrongBodyLabel</class>
      <extends>QLabel</extends>
      <header>qfluentwidgets</header>
  </customwidget>
     <customwidget>
         <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElevatedCardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
